### PR TITLE
Set the message processing concurrency in the monitoring instance

### PR DESCRIPTION
**Pull request by danielmarbach**
#2273 for release-4.13 branch

- Set the message processing concurrency in the monitoring instance
- Make the MaximumConcurrencyLevel level configurable

### CODE REVIEW

File: src/ServiceControl.Monitoring/Settings.cs [hunk 1]
```diff
@@ -8,16 +8,34 @@ namespace ServiceControl.Monitoring
...
+    using ServiceControl.Monitoring.Infrastructure.Settings;
 
     //TODO: align names with SC and SC.Audit
     public class Settings
     {
+        public Settings()
+        {
+            TryLoadLicenseFromConfig();
+
+            TransportType = SettingsReader<string>.Read("TransportType");
```
Comment (timbussmann) [inline src/ServiceControl.Monitoring/Settings.cs:20]: the removed code used `TransportType = reader.Read<string>("Monitoring/TransportType"),` with the monitoring prefix (some other options seem to do that too), is that an intentional change?

File: src/ServiceControl.Monitoring/Settings.cs [hunk 2]
```diff
@@ -59,6 +60,11 @@ internal static string DefaultLogLocation()
             return Path.GetDirectoryName(assemblyLocation);
         }
 
+        void TryLoadLicenseFromConfig()
```
Comment (timbussmann) [inline src/ServiceControl.Monitoring/Settings.cs:63]: not sure how this connects to the other changes.

File: src/ServiceControl.Monitoring/Settings.cs (hunk 1 above)
Comment (danielmarbach) [reply src/ServiceControl.Monitoring/Settings.cs]: Yeah these were changes that came in from 

https://github.com/Particular/ServiceControl/commit/cac7201349f4af9ffffa8f81aa2d19403fb2c007#diff-b616500841627c3849f0e83080ac2d844bbfdf95abd9f255bb7f6129c674fb12

Force pushed

**Review by WilliamBZA — APPROVED** [review]

**Comment by danielmarbach** [comment]
Was missing https://github.com/Particular/ServiceControl/pull/2266 to actually get compiling

**Review by timbussmann — APPROVED** [review]

**Pull request merged**